test/typed_wrapper/stream_test.dart

```diff
@@ -2,6 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+@Skip("Re-enable this when test can run DDC (test#414).")
+
```

Comment:
Please add a library tag, so the annotation applies to that, not the import below. I know that it "works", but it's still ugly. 

(We should allow an empty `library;` declaration for that use-case).

Reply:
That's contrary to the style of every other package that uses test annotations.

Reply:
Keep it then, until we make `library;` possible.